gemrb/GUIScripts/pst/GUIREC.py

```diff
@@ -848,11 +859,13 @@ def AcceptLevelUp():
 		GemRB.SetPlayerStat (pc, IE_LEVEL, GemRB.GetPlayerStat (pc, IE_LEVEL)+NumOfPrimLevUp)
 		if avatar_header['SecoLevel'] != 0:
 			GemRB.SetPlayerStat (pc, IE_LEVEL2, GemRB.GetPlayerStat (pc, IE_LEVEL2)+NumOfSecoLevUp)
-	
-	LUSkillsSelection.SkillsSave (pc)
 
-	LevelUpWindow.Close()
-	NewLife.OpenLUStatsWindow()
+	if LevelUpWindow:
```

Comment:
Can this really be false?

Reply:
No, but it's just part of a work in progress merge - I'm inching toward fully eliminating the need for the separate PST level up logic, sometimes I shuffle things around on a branch so they are easier to compare side by side. It's done like that in LevelUp.py, but I guess it doesn't necessarily have to be there either.